content/spin/v2/manifest-reference.md

```diff
@@ -144,7 +144,7 @@ The value of each key is a table with the following fields.
 |                         |            | Table       | * A file or directory to be made available to the Wasm module at a specific path. This must be a table containing a `source` field for the file or directory relative to the manifest file, and a `destination` field containing the absolute path at which to make it available. | `{ source = "assets/images", destination = "/pictures" }` |
 | `exclude_files`         | Optional   | Array of strings | Any files or glob patterns that should _not_ be available to the Wasm module at runtime, even though they match a `files` entry. | `[assets/images/test/**/*.*]` |
 | `allowed_http_hosts`    | Optional   | Array of strings | The host names or addresses to which the Wasm component is allowed to send HTTP requests. If the name includes a port, the Wasm module can send requests only to that port; otherwise, the Wasm module can send requests only to the default port for the scheme it uses. The special string `insecure:allow-all` permits the module to send HTTP requests to _any_ host, but is intended for development use only; some deployment environments may decline to honour it. | `["example.com", "localhost:8081"]` |
```

Comment:
```suggestion
| `allowed_http_hosts`    | Deprecated Optional   | Array of strings | Deprecated in favor of `allowed_outbound_hosts`. The host names or addresses to which the Wasm component is allowed to send HTTP requests. If the name includes a port, the Wasm module can send requests only to that port; otherwise, the Wasm module can send requests only to the default port for the scheme it uses. The special string `insecure:allow-all` permits the module to send HTTP requests to _any_ host, but is intended for development use only; some deployment environments may decline to honour it. | `["example.com", "localhost:8081"]` |
```

I very much think we should mark this as deprecated and perhaps move it to the end of the list. I think it would be unfortunate if new users find this documentation and mistakenly think that this is the right option to use.